<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoronoiForm</class>
 <widget class="QWidget" name="VoronoiForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="labelAlignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="ToolSelectorForm" name="toolHolder" native="true"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_7">
     
     <property name="text">
      <string>Solver:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbxSolver">
     <item>
      <property name="text">
       <string>BOOST</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>JC_VORONOI</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     
     <property name="text">
      <string>Precision:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="DoubleSpinBox" name="dsbxPrecision">
     <property name="suffix">
      <string> mm</string>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_8">
     
     <property name="text">
      <string>Frame Offset:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="DoubleSpinBox" name="dsbxOffset">
     <property name="suffix">
      <string> mm</string>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     
     <property name="text">
      <string>Width:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="DoubleSpinBox" name="dsbxWidth">
     <property name="suffix">
      <string> mm</string>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">doublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>ToolSelectorForm</class>
   <extends>QWidget</extends>
   <header location="global">tool_selectorform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
